projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -854,6 +855,8 @@ class ProjectorServer private constructor(
 
       SettingsInitializer.addTaskToInitializeIdea(PGraphics2D.defaultAa)
 
+      configureUpdates(isAgent)
```

Comment:
Let's make it configurable by introducing one more option (that will be `true` by default). Maybe some users will want to disable it.
Please don't forget to document it a bit: https://jetbrains.github.io/projector-client/mkdocs/latest/ij_user_guide/server_customization/